STEERING-COMMITTEE.md

```diff
@@ -162,11 +146,172 @@ There are two ways to raise issues to the steering committee for decision:
 2. Open an issue on a knative repository and indicate that you would like
    attention from the steering committee.
 
+## Composition
+
+The steering committee has 7 seats.  Five of the seats are Contributing
+seats, open to any project contributor.  Two of the seats are End-User
+seats, open only to active end-user community members.  See [candidate
+eligibility](#candidate-eligibility) for definitions of these categories.
+
+Steering Committee members serve for 2-year terms, staggered in order to
+preserve continuity.  Every year either two or three contributor seats are
+elected, and one end user seat is elected.
+
+## Election Procedure
```

Comment:
When will the first election be held if this proposal is adopted?

Reply:
It's on L283, October 2020

Reply:
Thanks, brain-melt happening for me these days